streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -212,6 +218,24 @@
                                         CommonClientConfigs.METRICS_NUM_SAMPLES_DOC);
     }
 
+    // this is the list of configs for underlying clients
+    // that streams prefer different default values
+    private static final Map<String, Object> PRODUCER_OVERRIDES;
```

Comment:
Nit: maybe we should call this `PRODUCER_DEFAULT_OVERRIDES` and the other one `CONSUMER_DEFAULT_OVERRIDES`.